FirebaseMLModelDownloader/Tests/Unit/ModelDownloaderUnitTests.swift

```diff
@@ -254,39 +254,32 @@ class NetworkingUnitTests: XCTestCase {
   }
 
   func testModelDownloadWith200() {
```

Comment:
nit: Usually we try to have a separate file per class being tested with all relevant test cases, e.g. ModelDownloadTaskTests.swift, ModelInfoRetrieverTest.swift, ModelDownloaderTests.swift, etc. (see [Installations](https://github.com/firebase/firebase-ios-sdk/tree/master/FirebaseInstallations/Source/Tests/Unit) as an example). It may make sense to split this file to several class related ones.

Reply:
Gotcha, thanks for the suggestion! Added a TODO.